tests/test_model_helpers/activations/test___init__.py

```diff
@@ -223,6 +285,13 @@ def test_exact_activations(pca_components):
                                        image_name='rgb.jpg', pca_components=pca_components, logits=False)
     path_to_expected = Path(__file__).parent / f'alexnet-rgb-{pca_components}.nc'
     expected = xr.load_dataarray(path_to_expected)
+
+    # Originally, the `stimulus_path` Index was used to index into xarrays in Brain-Score, but this was changed
+    #  as a part of PR #492 to a MultiIndex to allow metadata to be attached to multiple repetitions of the same
+    #  `stimulus_path`. Old .nc files need to be updated to use the `presentation` index instead of `stimulus_path`,
+    #  and instead of changing the extant activations, this test was simply modified to simulate that.
+    expected = expected.rename({'stimulus_path': 'presentation'})
+
     assert (activations == expected).all()
 
 
```

Comment:
I expected a test here that tests the exact values for `microsaccade_shift_x_pixels`, `microsaccade_shift_y_pixels`, `microsaccade_shift_x_degrees`, `microsaccade_shift_y_degrees`. The function `test_require_variance_has_shift_coords` only tests the rather trivial number of values, but not that the values are set correctly